src/components/PriceAlertForm/PriceAlertForm.tsx

```
@@ -18,6 +20,7 @@ import { Spinner } from '../ui/spinner';
 import { useToast } from '@/lib/contexts/toast-context';
 import { useCoinAndExchangeData } from '@/lib/hooks/useCoinAndExchangeData';
 import { useFormSubmission } from '@/lib/hooks/useFormSubmission';
+import { ConfirmDialog } from '../ui/confirm-dialog';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ConfirmDialog'. Consider to delete it or rename it.

'ConfirmDialog' is defined here:


(lint/suspicious/noRedeclare)